### PR TITLE
scripts/prepare-release: fix upating of readme

### PR DESCRIPTION
**Pull request by marquiz**
Fix the sed script to update the version of kustomize overlay.

Also, add a sed script to update the documentation link to point to the
correct version in gh-pages.

### CODE REVIEW

File: scripts/prepare-release.sh [hunk 1]
```diff
@@ -93,7 +93,9 @@ if [ -z "$assets_only" ]; then
 
     # Patch README
     echo Patching README.md to refer to $release
-    sed s"!node-feature-discovery/v.*/!node-feature-discovery/$release/!" -i README.md
+    sed -e s"!\(node-feature-discovery/deployment/.*\)=v.*!\1=$release!" -i README.md \
+        -e s"!^\[documentation\]:.*![documentation]: https://kubernetes-sigs.github.io/node-feature-discovery/$docs_version!" \
+        -i README.md
```
Comment (ArangoGutierrez) [inline scripts/prepare-release.sh:98]: I see README 2 times, is this desired?

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

File: scripts/prepare-release.sh (hunk 1 above)
Comment (marquiz) [reply scripts/prepare-release.sh:98]: Definitely not, good catch, thanks. Fixed

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**